README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
I think the Fixnum is incorrect there; I just went through the commit history and Fixnum has never had a valid usage for that option. I think the thought at one point might have been to have a sweeper thread which closed open connections older than X seconds, but that never materialized. In any case, it's generally the server's case to manage keepalive connections, so I'm going to just fix the doc to indicate that a boolean should be passed.